etc/non-ecs-schema.json

```diff
@@ -9,5 +9,8 @@
   },
   "winlogbeat-*": {
     "winlog.event_data.OriginalFileName": "keyword"
+  },
+  "filebeat-*": {
+    "o365.audit.Parameters.AllowFederatedUsers": "keyword"
```

Comment:
Dependent on the resolution from https://github.com/elastic/detection-rules/pull/578#issuecomment-740132064

Anything under `Parameters` or any other `X.*` of type object

If no resolution by COB, we should just merge this with the exception and resolve in the feature branch